app/src/main/java/no/usn/mob3000/ui/screens/user/ProfileEditScreen.kt

```diff
@@ -181,4 +181,18 @@ fun ProfileEditScreen(
             ) { Text(stringResource(R.string.profile_edit_save_button)) }
         }
     }
+    if (showDialog.value) {
+        profileConfirmDialog(
+            showDialog = showDialog,
+            onConfirm = {
+                onSaveProfileClick(displayName, avatarUrl, aboutMe, selectedCountry, isProfileVisible, isFriendsListVisible)
+                navigateToProfile()
+            },
+            onDismiss = {
+
+                showDialog.value = false
+            },
+            "Save changes?","do you want to save your changes?","Save","Cancel"
```

Comment:
Indentation. Additionally, if these are used in a `Composable`, then they should be extracted to string resources. Additionally, some may be incorrectly capitalized.